Dockerfile

```diff
@@ -8,10 +8,14 @@ COPY package.json /app
 COPY yarn.lock /app
 RUN yarn --frozen-lockfile && chown -R subquery /app
 
+########################################################################################
+# Following parameters are needed to generate a project.yaml from project.template.yaml
 ENV START_BLOCK=1
-# CALL_HANDLER and EVENT_HANDLER are needed to generate a project.yaml from project.template.yaml
 ENV CALL_HANDLER='callHandler'
 ENV EVENT_HANDLER='handleEvent'
+# ENV NETWORK_ENDPOINT='ws://host.docker.internal:9944'
+# ENV NETWORK_CHAIN_ID='0xda7f2072787bfd0b09f7e12fca619afb6041b3d620f39f3a508814869100bf01'
```

Comment:
should these be uncommented?

Could be nice to have a line about how to get chain id, or if it defaults to the right thing.

Reply:
Yes, I completely forgot about this. So the point was I was trying to re-create the network section within the project.yaml file as we create the handlers(using $CALL_HANDLER and $EVENT_HANDLER).So I introduced these vars so that would just pass them in the docker file, but apparently when I try to do this, the validations within the subquery fails stating chainId not found, even though the project.yaml file is perfectly created using the envsubst.

Can you try it out once Eric? I'm not that good with docker. Maybe I missed something. I was hoping to achieve a network section as follows - 
```
network: 
  chainId: ${NETWORK_CHAIN_ID}
  endpoint: ${NETWORK_ENDPOINT}
```

Reply:
https://github.com/subquery/subql/blob/main/packages/common-substrate/src/project/versioned/v1_0_0/model.ts#L64 This is where it fails stating something like - cannot find value.trim()

Reply:
Thanks for the input